src/main/java/org/ibp/api/rest/labelprinting/LabelPrintingResource.java

```diff
@@ -116,6 +120,20 @@
 		return new ResponseEntity<>(labelTypes, HttpStatus.OK);
 	}
 
+	@RequestMapping(value = "/crops/{cropname}/programs/{programUUID}/labelPrinting/{labelPrintingType}/SortableFields", method = RequestMethod.GET)
```

Comment:
rename by sortable-fields
We still have some problems with the name of some resources, they should not be camelCase. 
I will create a ticket to rename all the label printing services to use label-printing instead of labelPrinting